db/seeds.rb

```diff
@@ -27,11 +28,18 @@ def create_beer
 
   def create_users
     users.each do |full_name, email, pw, role, phone, display_name|
-      User.create(fullname: full_name, email: email, password_digest: pw, role: role, phone: phone, display_name: display_name)
+      User.create(fullname: full_name, email: email, password: pw, role: role, phone: phone, display_name: display_name)
     end
     puts "Users: #{User.all.map(&:fullname).join(", ")} created."
   end
 
+  def create_categories
+    categories.each do |name, description|
+      Category.create(name: name, description: description)
+    end
+    puts "Category: #{Category.all.map(&:name).join(", ")} created."
```

Comment:
Prefer single-quoted strings inside interpolations.<br>Do not write to stdout. Use Rails' logger if you want to log.